roles/k3s/master/tasks/main.yml

```diff
@@ -64,8 +64,7 @@
     cmd: "systemd-run -p RestartSec=2 \
                       -p Restart=on-failure \
                       --unit=k3s-init \
-                      k3s server {{ server_init_args }} \
-                      {{ '--node-taint CriticalAddonsOnly=true:NoExecute' if k3s_single_node|bool == false else ''}}"
+                      k3s server {{ server_init_args }}"
```

Comment:
How is this related to your change?

Reply:
It's not, it's due to what I just sent related to my bad approach to this / my first pull request.  100% my fault.  But - frankly, -i'm not sure I want this for my use case, so I'm not sure If I want it applied to all machines.  my intent is to run longhorn on all nodes along with workloads as this is totally home-lab driven.  Shouldn't we let people put server args in the all.yaml in group_vars?  Maybe more of a comment there?